ydb/_grpc/grpcwrapper/common_utils.py

```diff
@@ -149,12 +151,17 @@ class GrpcWrapperAsyncIO(IGrpcWrapperAsyncIO):
     convert_server_grpc_to_wrapper: Callable[[Any], Any]
     _connection_state: str
     _stream_call: Optional[Union[grpc.aio.StreamStreamCall, "grpc._channel._MultiThreadedRendezvous"]]
+    _wait_executor: Optional[concurrent.futures.ThreadPoolExecutor]
 
     def __init__(self, convert_server_grpc_to_wrapper):
         self.from_client_grpc = asyncio.Queue()
         self.convert_server_grpc_to_wrapper = convert_server_grpc_to_wrapper
         self._connection_state = "new"
         self._stream_call = None
+        self._wait_executor = None
+
+    def __del__(self):
+        self._wait_executor.shutdown(wait=False)
```

Comment:
do we need to check `_wait_executor` here too? or use `_clean_executor` instead

Reply:
Thanks, fixed